lib/user_interface.rb

```diff
@@ -7,6 +7,10 @@ def display_message(message)
     @platform.display_message message
   end
 
+  def display_empty_line
+    @platform.display_empty_line
```

Comment:
Could messages just be assumed to have empty lines at the end of their strings?

Reply:
Or another thing to think about, is an empty line just another message?

Reply:
Olga asked me something related, so I will copy the comment I left:

"When working on different platforms, I thought empty lines could be ignored. For example, on web, there is no need to have an empty line, because that can be dealt with in CSS."

https://github.com/k3ntako/super-tic-tac-toe/pull/16#discussion_r390434746